lib/opencensus/trace/formatters/binary.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017 OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module OpenCensus
+  module Trace
+    module Formatters
+      ##
+      # This formatter serializes and deserializes span context according to
+      # the OpenCensus' BinaryEncoding specification. See
+      # [documentation](https://github.com/census-instrumentation/opencensus-specs/blob/master/encodings/BinaryEncoding.md).
+      #
+      class Binary
+        BINARY_FORMAT = "CCH32CH16CC".freeze
```

Comment:
Put a `## @private` in front of these internal constants to signal to yardoc that these are uninteresting.

Same with the other two formatters.